deploy/examples/radosnamespace.yaml

```diff
@@ -9,3 +9,14 @@ spec:
   # name: namespace-a
   # blockPoolName is the name of the CephBlockPool CR where the namespace will be created.
   blockPoolName: replicapool
+  mirroring:
```

Comment:
How about if we create a new radosnamespace-mirrored.yaml that has the new example where it is `enabled: true`? Mirroring is really an advanced feature. Then the main example for rados namespace can remain very simple.